### DISCUSSION TITLE
Alcoholic distillates

### DISCUSSION THREAD

**Discussion by snow-lumi** [yuorb/lexicon-json#13] [Ideas]
The current wording of the roots for alcoholic beverages seems to exclude drinks made by distillation like cognac, schnaps, whiskey or amaretto.
I'd suggest to add either one root, equivalent to the word liquor/烈酒 or to split it up into fruit/grain/other like for fermented drinks.
Not to be confused with liqueurs, which are typically mixtures or infusions.

**Comment by ryanlo713** [comment]
Do you want to add one or more roots, or supplement the usage of existing roots?

If you need to add more, it is recommonded to provide the design of stem and specification.

If it is a supplement, indicate the root from which the definition can be expanded.
